src/main/java/org/kitteh/irc/client/library/implementation/NettyManager.java

```diff
@@ -286,18 +305,108 @@ public String toString() {
         }
     }
 
+    static class DCCConnection extends ChannelInitializer<SocketChannel> {
+
+        private final IRCDCCExchange exchange;
+        private final InternalClient client;
+        // Only allow one connection per DCC. Weird, I know.
+        private boolean oneConnection;
```

Comment:
Perhaps rename to `connected`? Or something similar. `if(oneConnection)` reads oddly to me.